src/BibleWell.App/Views/MainView.axaml.cs

```diff
@@ -9,6 +11,30 @@ public partial class MainView : UserControl
     public MainView()
     {
         InitializeComponent();
+        DataContextChanged += (_, _) =>
+        {
+            if (DataContext is MainViewModel vm)
+            {
+                vm.PropertyChanged += Vm_PropertyChanged;
+            }
+        };
+    }
+
+    private void Vm_PropertyChanged(object? sender, PropertyChangedEventArgs e)
+    {
+        // Activate the line above the menu
+        if (sender is MainViewModel vm && e.PropertyName == nameof(vm.NavMenuVisible))
+        {
+            BottomNavBorder.BorderThickness = vm.NavMenuVisible ? new Thickness(0, 1, 0, 0) : new Thickness(0);
```

Comment:
Can you use style class binding in the axaml instead of this property changed event? https://docs.avaloniaui.net/docs/guides/data-binding/binding-classes